.github/workflows/changelog_verifier.yml

```diff
@@ -0,0 +1,19 @@
+name: "Changelog Verifier"
+on:
+  pull_request:
+    types: [opened, edited, review_requested, synchronize, reopened, ready_for_review, labeled, unlabeled]
```

Comment:
Do we want to filter based on branch?   (I think probably yes.)

Reply:
Good idea. Added it.